src/Storage/RegressionTests/srp_preview.ps1

```diff
@@ -34,6 +34,119 @@ Describe "Management plan test - preview" {
 
         $Error.Count | should -be 0
     }
+
+    It "LCM TierTo Cold, TierToHot" {
```

Comment:
(Optional) We can set TierToHot, TierToCool together, to save the test time.

Reply:
Will combine them later. Was having some trouble creating premium blob storage account under region eastus2euap, which is needed if we want to test both TierToCold and TierToHot at the same time.